appsec/_test/go.mod

```diff
@@ -0,0 +1,13 @@
+module appsectest
```

Comment:
What was the reasoning behind this? This should not be a separate module. It needs not be released to the public separately from the rest, right?

Reply:
As explained above, this is a small overall validation test so it uses the tracer and I don't want to have it in the appsec go.mod file only because of that useful test.

Reply:
Module now removed in 2259fa6dd2c9ad6c2a310125f68a2e91a0e011df